cogs/twitcasting/__init__.py

```diff
@@ -0,0 +1,23 @@
+import threading
+
+from .twitcasting import *
+from aiohttp import web
+import asyncio
+
+
+
+def setup(bot):
+    twitcasting_cog = Twitcasting(bot)
+    bot.add_cog(twitcasting_cog)
+
+    app = web.Application()
+    app.router.add_post('/', twitcasting_cog.broadcast)
```

Comment:
/twitcasting route please 
TODO: move out of this directory, manga webhooks incoming